examples/lfric/eg17/full_example/Makefile

```diff
@@ -1,7 +1,7 @@
 # -----------------------------------------------------------------------------
 # BSD 3-Clause License
 #
-# Copyright (c) 2020, Science and Technology Facilities Council.
+# Copyright (c) 2020-2021, Science and Technology Facilities Council.
```

Comment:
I tried to compile this example with the Intel 17.0.1 compiler like this:
```
F90=ifort F90FLAGS="-g -check bounds" make clean
```
It failed with
```
testkern_w0_mod.f90(67): error #6683: A kind type parameter must be a compile-time constant.   [I_DEF]
    integer(kind=i_def)                                 :: ndf_w0, undf_w0
-----------------^
testkern_w0_mod.f90(68): error #6683: A kind type parameter must be a compile-time constant.   [R_DEF]
    real(kind=r_def), dimension(undf_w0), intent(inout) :: fld1
--------------^
testkern_w0_mod.f90(69): error #6683: A kind type parameter must be a compile-time constant.   [R_DEF]
    real(kind=r_def), dimension(undf_w0), intent(in)    :: fld2
--------------^
testkern_w0_mod.f90(70): error #6683: A kind type parameter must be a compile-time constant.   [I_DEF]
    integer(kind=i_def), dimension(ndf_w0)              :: map_w0
-----------------^
testkern_w0_mod.f90(72): error #6683: A kind type parameter must be a compile-time constant.   [I_DEF]
    integer(kind=i_def)                                 :: i, k
-----------------^
testkern_w0_mod.f90(39): error #6580: Name in only-list does not exist.   [W0]
  use fs_continuity_mod, only: W0
-------------------------------^
testkern_w0_mod.f90(53): error #6404: This name does not have a type, and must have an explicit type.   [CELL_COLUMN]
     integer :: operates_on = cell_column
```
Then I explicitly exported `F90` and `F90FLAGS`:
```
export F90=ifort
export F90FLAGS="-g -check bounds"
```
and tried again with just `make compile`. It failed with the same error. it looks to me as if there is a problem with using the LFRic infrastructure library modules. Could you please investigate?

Reply:
It works for me (tm) :grin: But that's with intel 21
The only idea I have is that some files are compiled with a different compiler (to my surprise it looks like intel 21 at least read gfortran module files ???) I tried to reproduce on our Crays (the only platform where I have intel 17), but ... I could not update my psyclone installation due to the submodules (which I can't access thanks to firewalls). I tried to replace the whole extract tree with a right copy, but now pkg_resrouces is complaining about something missing @*(&#*@(&# Before I sort this out, can you try to do a:
```
make allclean
F90=ifort F90FLAGS="-g" make compile
```
And the also provide the fill compilation line with all options?

This is really strange, since no file was changed, this PR only moved them into a different directory. I can't see how the location affects compilation (assuming that the include path is correctly updated, but then we should also have an error about missing modules in the first place).

Reply:
@TeranIvy I still need your update on this :)